window_action.talon

```diff
@@ -1,12 +1,30 @@
 os: mac
 -
+
+# Targeting the windows of the current application (e.g. "window close other", "window minimize"):
+window {user.window_actions}: user.action_windows(user.window_actions, 1, 0)
+window {user.window_actions} other: user.action_windows(user.window_actions, 0, 1)
+window {user.window_actions} all: user.action_windows(user.window_actions, 1, 1)
+
+# Targeting the windows of any arbitrary application (e.g. "Finder window close all"):
+<user.running_applications> window {user.window_actions}:
+    user.action_windows(user.window_actions, 1, 0, user.running_applications)
+<user.running_applications> window {user.window_actions} other:
+    user.action_windows(user.window_actions, 0, 1, user.running_applications)
+<user.running_applications> window {user.window_actions} all:
+    user.action_windows(user.window_actions, 1, 1, user.running_applications)
```

Comment:
mixed feelings about having these start with app name.  seems like potentially a big capture to lead a command with.  Idk maybe that's fine?  here's a few alternatives:

- "from finder window close all"
- "app finder window close all"
- "window close all finder"

Reply:
Hmm that doesn't sound very noun-verb 🤔

I think the verb-nouniest would maybe be "window close all finder" or something like that

Reply:
Given how rarely I suspect it will be used, "window close all <app name>" makes sense to me too.